File: SikuliX/JAVA/win/7/highres/src/main/java/cura/objects/AppointmentForm.java
package cura.objects;

import org.sikuli.script.Pattern;

import java.util.ArrayList;
import java.util.List;

public class AppointmentForm {

    public static Pattern apply_readmission = new Pattern("imgs/apply_readmission.png").targetOffset(-107,-1);
    public static Pattern book = new Pattern("imgs/book.png");
    public static Pattern comment = new Pattern("imgs/comment.png").targetOffset(125,7);
    public static Pattern facility = new Pattern("imgs/facility.png").targetOffset(149,1);
    public static List<String> facility_values = new ArrayList<String>(){{
        add("Tokyo CURA Healthcare Center");
        add("Hongkong CURA Healthcare Center");
        add("Seoul CURA Healthcare Center");
    }};
    public static List<Pattern> healthcare_programs = new ArrayList<Pattern>(){{
        add(new Pattern("imgs/medicare.png"));
        add(new Pattern("imgs/medicaid.png"));
        add(new Pattern("imgs/none.png"));
    }};
    public static Pattern title = new Pattern("imgs/make_appointment_title.png");
    public static Pattern visit_date = new Pattern("imgs/visit_date.png").targetOffset(168,-4);
}

File: WEB/Nightwatch/osx/objects/AppointmentForm.js
module.exports = {

    commands : {

        create_appointment : function(apply, comment, facility, healtcare_program, visit_date) {
            console.log("Appointment Creation ...");
            // ToDo
        }
    },

    elements : {

        apply_readmission : {
            selector : "#chk_hospotal_readmission"
        },
        book : {
            selector : "#btn-book-appointment"
        },
        comment : {
            selector : "#txt_comment"
        },
        facillity : {
            selector : "#combo_facility",
            values : [
                "Tokyo CURA Healthcare Center", 
                "Hongkong CURA Healthcare Center",
                "Seoul CURA Healthcare Center"
            ]
        },
        healthcare_programs : {
            medicare : {
                selector : "#radio_program_medicare"
            },
            medicaid : {
                selector : "#radio_program_medicaid"
            },
            none : {
                selector : "#radio_program_none"
            }
        },
        title : {
            selector : "h2",
            text : "Make Appointment"
        },
        visit_date : {
            selector : "#txt_visit_date"
        }
    },

    url : "http://demoaut.katalon.com/"
}